app/static/js/app.js

```diff
@@ -247,15 +250,50 @@ function onKeyDown(evt) {
 }
 
 function sendMouseEvent(evt) {
-  const boundingRect = evt.target.getBoundingClientRect();
-  const cursorX = Math.max(0, evt.clientX - boundingRect.left);
-  const cursorY = Math.max(0, evt.clientY - boundingRect.top);
+  // Ensure that e.g. mouse drags don't attempt to drag the image on the screen.
+  evt.preventDefault();
+
+  emitMouseEvent(evt.target, { x: evt.clientX, y: evt.clientY }, evt.buttons);
+}
+
+function sendTouchEvent(evt) {
+  // Ensure that drags don't attempt to drag the image on the screen.
+  evt.preventDefault();
+
+  const touches = evt.touches;
+
+  if (touches.length == 0 && lastPosition !== undefined) {
```

Comment:
Could we add a comment explaining which case this is for? Can any touch event trigger this code snippet or is it for a subset of the events?